cobbler/utils.py

```diff
@@ -502,17 +502,15 @@ def file_is_remote(file_location) -> bool:
     return False
 
 
-def input_string_or_list(options: Union[str, list]) -> Union[list, str]:
+def input_string_or_list_no_inherit(options: Optional[Union[str, list]]) -> list:
     """
     Accepts a delimited list of stuff or a list, but always returns a list.
 
     :param options: The object to split into a list.
-    :return: str when this functions get's passed <<inherit>>. if option is delete then an empty list is returned.
+    :return: If ``option`` is ``delete`` or ``None``, then an empty list is returned.
              Otherwise this function tries to return the arg option or tries to split it into a list.
-    :raises TypeError: Raised in case the input type is wrong.
+    :raises TypeError: In case the type of ``options`` was neither None, str or list.
     """
-    if options == enums.VALUE_INHERITED:
-        return enums.VALUE_INHERITED
     if not options or options == "delete":
```

Comment:
This does not the same as the docstring says: `bool(options)` is falsy if it is `None`, `""` or `[]`

Reply:
Fixed via docstring correction.